big_tests/tests/mongoose_helper.erl

```diff
@@ -301,3 +307,27 @@ wait_for_user(Config, User, LeftTime) ->
                                  name => 'escalus_users:create_user'
                                 }).
 
+% Loads a module present in big tests into a MongooseIM node
+-spec inject_module(Module :: module()) -> ok.
+inject_module(Module) ->
+    inject_module(Module, false).
+
+-spec inject_module(Module :: module(), CheckIfLoadedAlready :: boolean()) -> ok | already_loaded.
```

Comment:
I feel like `DoOverwrite` would better explain what is its role

Reply:
OK, will pick a better name

Reply:
Boolean params are evil. If you don't know the definition by heart, what does this call site tell you?

```erlang
inject_module(Module, false)
```

Inject if `true`, do not if `false` - so do not really inject the module here? Inject, but `false` what?
Instead: `inject_module(Module, force_reload)`, or `inject_module(Module, do_overwrite)`, or `inject_module(Module, [force_reload])`...